system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/tracing/utils/constants.js

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
Why this is removed?

Reply:
We use the constants from [https://github.com/Shairam/product-vick/blob/master/system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/common/constants.js](https://github.com/Shairam/product-vick/blob/master/system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/common/constants.js)